Test/PcapDeviceTest.cs

```diff
@@ -34,12 +34,21 @@ namespace Test
     [NonParallelizable]
     public class PcapDeviceTest
     {
+        /// <summary>
+        /// Test that we can convert each TimestampType
+        /// </summary>
         [Test]
-        public void DeviceProperties([PcapDevices] DeviceFixture fixture)
+        public void TimestampConversions()
         {
-            using var device = (PcapDevice)fixture.GetDevice();
+            foreach(TimestampType timestampType in Enum.GetValues(typeof(TimestampType)))
+            {
+                Assert.IsNotEmpty(PcapDevice.TimestampName(timestampType));
+                Assert.IsNotEmpty(PcapDevice.TimestampDescription(timestampType));
```

Comment:
I think we could make the timestamp API more object oriented. Proposal:
Create class `PcapClock`, that describes and encapsulates `TimestampType`:
```cs
public class PcapClock {
    public string Name { get; }
    public string Description { get; }
    public TimestampPrecision Precision {get; } // High, Low, Unknown
    public bool? Synchronized { get; }
}
```
this provide easier distinction between different timestamp types.

Reply:
I think this makes sense. If we rename the existing precision to resolution what do we use for the types of resolution here?